src/Aspire.Hosting.SqlServer/SqlServerCreateDatabaseScriptAnnotation.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting;
```

Comment:
this should be `Aspire.Hosting.SqlServer`.

Reply:
I tried :/ but then there is a conflict with the SqlServer CDK class in another file. And I can fix it without doing a `global::namespace.SqlServer` for every usage. `using SqlServer = ... `  didn't work.